src/screens/send/types.ts

```diff
@@ -15,6 +15,7 @@ export interface TransactionInformation {
   value?: string
   symbol?: string
   hash?: string
+  original?: object
```

Comment:
This should be `Transaction` from ethers.

Reply:
Good to go - I've used the TransactionResponseWithoutWait